ce/api/util.py

```diff
@@ -80,3 +81,19 @@ def mean_datetime(datetimes):
     timestamps = [ dt.replace(tzinfo=timezone.utc).timestamp() for dt in datetimes ]
     mean = np.mean(timestamps)
     return datetime.fromtimestamp(mean, tz=timezone.utc)
+
+def search_for_unique_ids(sesh, ensemble_name='ce', model='', emission='', variable='', time=0):
+    query = sesh.query(DataFile.unique_id)\
+                  .join(DataFileVariable, EnsembleDataFileVariables, Ensemble, Run, Model, Emission)\
+                  .filter(Ensemble.name == ensemble_name)\
+                  .filter(DataFileVariable.netcdf_variable_name == variable)\
```

Comment:
A single file can potentially contain multiple variables (though it's not recommended), but only ever represents a single model and a single emission scenario.